src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -5,10 +5,12 @@ import IconDrawing24 from '../icons/IconDrawing24';
 import IconHighlightText16 from '../icons/IconHighlightText16';
 import IconRegion24 from '../icons/IconRegion24';
 import useFullscreen from '../hooks/useFullscreen';
+import { AnnotationColor } from '../../../AnnotationModule';
 import { AnnotationMode } from './types';
 import './AnnotationsControls.scss';
 
 export type Props = {
+    annotationColor?: string;
```

Comment:
Should this be `AnnotationColor` instead of string?

Reply:
AnnotationColor is an object rather than an enum so I think we need to use string here.

```
export const AnnotationColor = {
    BOX_BLUE: bdlBoxBlue,
    GREEN_LIGHT: bdlGreenLight,
    WATERMELON_RED: bdlWatermelonRed,
    YELLORANGE: bdlYellorange,
    YELLOW: bdlYellow,
    GRIMACE: bdlGrimace,
};
```

Reply:
It was an enum previously, which seems superior to a plain object. Why did we change it?

Reply:
We changed it because I was running into a typing issue where TS didn't like how I was trying to create an enum where the right hand assignment was an externally imported constant.

Here's what I tried before:
```
export enum AnnotationColor = {
     BOX_BLUE = bdlBoxBlue,
     ...
}
```

Reply:
I couldn't find any other instances in our codebases where we were able to achieve the desired effect so I opted for using an object instead.